numba_dpex/experimental/_kernel_dpcpp_spirv_overloads/_atomic_ref_overloads.py

```diff
@@ -69,6 +69,10 @@ def gen(context, builder, sig, args):
             "--spirv-ext=+SPV_EXT_shader_atomic_float_add"
         ]
 
+        context.extra_compile_options[LLVM_SPIRV_ARGS] = [
```

Comment:
Adding the extra flags here will cause you to hit https://github.com/IntelPython/numba-dpex/issues/1262. Please add the extension flag to spirv_generator as a temporary workaround.

Reply:
Added.